### PR TITLE
Consistent names for Transactions representing ASP.NET Core routes

### PR DESCRIPTION
**Pull request by jamescrosswell**
Resolves https://github.com/getsentry/sentry-dotnet/issues/2276

### CODE REVIEW

File: src/Sentry.AspNetCore/RouteUtils.cs [hunk 1]
```diff
@@ -35,7 +35,8 @@ internal static class RouteUtils
             builder.Append(routePattern);
         }
 
-        return builder.ToString();
+        // Force a leading slash (if there isn't already one present)
+        return $"/{builder.ToString().TrimStart('/')}";
```
Comment (bruno-garcia) [inline src/Sentry.AspNetCore/RouteUtils.cs:39]: hm this seems lazy, can we check if `/` is there instead of adding and trimming?

Comment (bruno-garcia) [inline src/Sentry.AspNetCore/RouteUtils.cs:39]: something like: 
```suggestion
        var url = builder.ToString();
        return url.Length>0 && url[0] == '/' ? url : $"/{url}";
```

Comment (jamescrosswell) [reply src/Sentry.AspNetCore/RouteUtils.cs:39]: Looks computationally less expensive

File: src/Sentry.AspNetCore/RouteUtils.cs [hunk 2]
```diff
@@ -35,7 +35,9 @@ internal static class RouteUtils
             builder.Append(routePattern);
         }
 
-        return builder.ToString();
+        // Force a leading slash (if there isn't already one present)
+        var url = builder.ToString();
+        return url.Length>0 && url[0] == '/' ? url : $"/{url}";
```
Comment (bruno-garcia) [inline src/Sentry.AspNetCore/RouteUtils.cs:40]:
```suggestion
        return url.Length >0 && url[0] == '/' ? url : $"/{url}";
```

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**